tests/simpleKVBC/TesterCRE/main.cpp

```diff
@@ -177,6 +177,38 @@ class PublicKeyExchangeHandler : public IStateHandler {
   }
 };
 
+class ClientsAddRemoveHandler : public IStateHandler {
+ public:
+  bool validate(const State& state) const override {
+    concord::messages::ClientReconfigurationStateReply crep;
+    concord::messages::deserialize(state.data, crep);
+    return std::holds_alternative<concord::messages::ClientsAddRemoveCommand>(crep.response);
+  }
+  bool execute(const State& state, WriteState& out) override {
+    LOG_INFO(getLogger(), "execute clientsAddRemoveCommand");
+    concord::messages::ClientReconfigurationStateReply crep;
+    concord::messages::deserialize(state.data, crep);
+    concord::messages::ClientsAddRemoveCommand command =
+        std::get<concord::messages::ClientsAddRemoveCommand>(crep.response);
+
+    concord::messages::ReconfigurationRequest rreq;
+    concord::messages::ClientsAddRemoveUpdateCommand creq;
+    creq.config_descriptor = command.config_descriptor;
+    rreq.command = creq;
+    std::vector<uint8_t> req_buf;
+    concord::messages::serialize(req_buf, rreq);
+    out = {req_buf, [this, &command]() {
```

Comment:
command is a variable on stack. i am not sure if we should capture this by reference in the lambda

Reply:
fixed